sdk/machinelearning/arm-machinelearning/samples/v2/javascript/batchDeploymentsCreateOrUpdateSample.js

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ *
+ * Code generated by Microsoft (R) AutoRest Code Generator.
+ * Changes may cause incorrect behavior and will be lost if the code is regenerated.
+ */
+
+// Copyright (c) Microsoft Corporation.
```

Comment:
Why do we have these changes? Does that mean we forgot to publish samples last time?